src/index.html

```diff
@@ -15,8 +15,78 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        class="logo"
+        src="/src/images/logo.png"
+        alt="Moyo logo"
+      />
+      <nav class="navbar">
+        <a
+          href="#"
```

Comment:
The `href` attribute for the navigation links is set to `#`, which is a placeholder. Ensure these links point to the correct sections or pages when the site is fully implemented.